bin/embedding-sdk/generate-sdk-package-json.js

```diff
@@ -68,6 +69,17 @@ function generateSdkPackage() {
     mergedContentString,
     "utf-8",
   );
+
+  const licenseContent = fs.readFileSync(
+    path.resolve(`./enterprise/LICENSE.txt`),
+    "utf-8",
+  );
+
+  fs.writeFileSync(
+    path.resolve(path.join(sdkDistDirPath), "LICENSE.txt"),
+    licenseContent,
+    "utf-8",
+  );
 }
 
 generateSdkPackage();
```

Comment:
I know this is still related to generating the package json in a broad sense, but I have a feeling we should split this function in two (generate json file, copy the license) and possibly renaming the script to a more generic name.
Otherwise looks good, I tested and it works

Reply:
Good catch! Let me clean this up a bit 👍🏻